src/DataTable/tests/BulkActions.test.jsx

```diff
@@ -82,46 +82,63 @@ describe('<BulkActions />', () => {
   describe('with functional rendering', () => {
     it('renders the function', () => {
       const myFunction = () => <Button>Some Button</Button>;
-      const wrapper = mount(<BulkActionsWrapper value={{ ...instance, bulkActions: myFunction }} />);
-      const button = wrapper.find(Button);
-      expect(button.length).toEqual(1);
+      const { container } = render(<BulkActionsWrapper value={{ ...instance, bulkActions: myFunction }} />);
+      const button = within(container).getByText('Some Button');
```

Comment:
Is there difference between `within(container).getByText` and `container.getByText`?